spacy/language.py

```diff
@@ -1597,8 +1601,21 @@ def _multiprocessing_pipe(
         n_process: int,
         batch_size: int,
     ) -> Iterator[Doc]:
+        def prepare_doc(
```

Comment:
I'm still not happy with my own naming suggestion. `prepare_input`? `split_context`?

Reply:
I'll go for `prepare_input`. The comment inside the function should describe the intent more concretely, in any event.